package.json

```diff
@@ -1,17 +1,17 @@
 {
   "devDependencies": {
     "react-native-get-random-values": "1.4.0",
-    "react-native-safe-area-context": "4.6.3",
-    "react-native-screens": "3.22.0",
-    "react-native-svg": "13.9.0",
-    "react-native-webview": "11.26.1",
-    "@react-native-masked-view/masked-view": "0.2.9",
+    "react-native-safe-area-context": "4.8.2",
+    "react-native-screens": "3.29.0",
+    "react-native-svg": "14.0.0",
+    "react-native-webview": "13.6.1",
+    "@react-native-masked-view/masked-view": "0.3.0",
     "@react-native-clipboard/clipboard": "1.11.2",
     "react-native-fast-image": "8.5.11",
-    "react-native-reanimated": "2.17.0",
-    "react-native-gesture-handler": "2.10.2",
+    "react-native-reanimated": "3.6.2",
+    "react-native-gesture-handler": "2.14.1",
     "react-native-linear-gradient": "2.7.3",
-    "react-native": "0.71.11"
+    "react-native": "0.73.3"
```

Comment:
>How were the versions proposed here selected?

For `react-native-screens` the `3.28.0` version added [support for React Native 0.73](https://github.com/software-mansion/react-native-screens/releases/tag/3.28.0). The newer `3.29.0` version fixed an issue from `3.28.0`. 

For `react-native-gesture-handler` I just picked the latest available version, I can change it to `2.14.0` Although I saw `2.14.1` includes a fix for a bug happening [in RN 0.73](https://github.com/software-mansion/react-native-gesture-handler/pull/2715) I'm not sure if it would affect our project since it also mentions Expo, but maybe it might be worth including it, what do you think?

Reply:
Thanks for sharing the rationale for the selections. I believe it is best to utilize the newer versions. We'll just need to update our Gutenberg core versions to match exactly. 👍🏻